lib/components/molecules/RenderTable/RenderTable.js

```diff
@@ -0,0 +1,51 @@
+// @flow
+/**
+ *
+ * RenderTable
+ *
+ */
+import React, { cloneElement, Children } from 'react';
+import styled from 'styled-components';
+
+import styles from './RenderTable.style';
+import type { renderTableProps } from './types';
+
+const RenderTable = ({ children, data, className, ...props }: renderTableProps) => {
+  return (
+    <table className={className} {...props}>
+      <thead>
+        <tr>
+          {Children.map(children, column => (
+            <th key={column.props.name}>{cloneElement(column, { intent: 'header' })}</th>
```

Comment:
First of all, the React.cloneElement example only works if your child is a single React element. 

I don't see a use-case of cloneElement